### PR TITLE
Changes per new cross-origin prefetch page.

### PR DESCRIPTION
**Pull request by twifkak**
Simplify the aside text by linking to the explanation on the new landing page,
and add a comparison to the cross-site prefetching space, per the promise on
that page. Also clarify a few other sentences while I'm here.

/cc @malchata @sidlall

### CODE REVIEW

File: src/site/content/en/fast/signed-exchanges/index.md [hunk 1]
```diff
@@ -325,6 +305,46 @@ Any site can cache, serve, and prefetch SXGs of the pages that it links to, wher
 ```
 [This article](/how-to-distribute-signed-http-exchanges/) illustrates how to use nginx to distribute SXGs.
 
+## Unique Advantages
+
+SXG is one of many possible technologies to enable [cross-origin prefetching](https://developer.chrome.com/blog/cross-origin-prefetch/). When deciding which technology to use, you may need to trade off between optimizing different aspects. The following sections illustrate a few of the unique values that SXG provides in the space of possible solutions.
```
Comment (sidlall) [inline src/site/content/en/fast/signed-exchanges/index.md:310]:
```suggestion
SXG is one of many possible technologies to enable cross-origin prefetching When deciding which technology to use, you may need to trade off between optimizing different aspects. The following sections illustrate a few of the unique values that SXG provides in the space of possible solutions.
```

File: src/site/content/en/fast/signed-exchanges/index.md [hunk 2]
```diff
@@ -325,6 +305,46 @@ Any site can cache, serve, and prefetch SXGs of the pages that it links to, wher
...
+
+### Fewer requests to serve
+
+With cross-site prefetching, your server may need to serve additional requests. This corresponds to cases where a page was prefetched, but either the user didn't visit the page, or the prefetched bytes couldn't be shown to the user. For SXG, this additional traffic can be significantly reduced:
```
Comment (sidlall) [inline src/site/content/en/fast/signed-exchanges/index.md:314]: Given that the audience is both websites and CDN partners,, what do you think about framing this along the lines of fewer unnavigated/ wasted requests to serve. Also calling out that over time we expect a better user experience from improved performance to mean more actual navigations to sites

File: src/site/content/en/fast/signed-exchanges/index.md [hunk 3]
```diff
@@ -325,6 +305,46 @@ Any site can cache, serve, and prefetch SXGs of the pages that it links to, wher
...
+
+- SXGs are cached and may be sent to users until they expire.
+- SXGs can be shown to users with cookies on your site.
```
Comment (sidlall) [inline src/site/content/en/fast/signed-exchanges/index.md:317]: Not sure I follow how this helps with reducing serves. Also repeated 2 sentences later

File: src/site/content/en/fast/signed-exchanges/index.md [hunk 4]
```diff
@@ -325,6 +305,46 @@ Any site can cache, serve, and prefetch SXGs of the pages that it links to, wher
...
+In the long term, other factors may influence which technology will provide the biggest improvement to page speed.
+{% endDetailsSummary %}
+
+- As more referrers implement cross-site prefetching for their outlinks, they may choose to implement only one technology, or multiple.
+- The reduced request serving for publishers could enable referrers to prefetch SXGs for links with lower predicted clickthrough rate.
+{% endDetails %}
+
+## Roadmap
+
+Google Search is continuing to improve the functionality of its SXG cache. Here are a couple upcoming improvements that address common limitations for sites looking to implement SXG.
```
Comment (sidlall) [inline src/site/content/en/fast/signed-exchanges/index.md:339]: Does this section need to be framed in the context of Google Search? i.e. If another referrer (internal or external) were to adopt SXG, would these roadmap features not benefit them as well?

File: src/site/content/en/fast/signed-exchanges/index.md [hunk 5]
```diff
@@ -325,6 +305,46 @@ Any site can cache, serve, and prefetch SXGs of the pages that it links to, wher
...
+
+- SXGs are cached and may be sent to users until they expire.
+- SXGs can be shown to users with cookies on your site.
+
+### Page speed improvement
+
+You may see additional page speed improvement from the following factors:
+
+- SXGs can be shown to users with cookies for your site.
+- SXG can also prefetch subresources for your pages, such as JavaScript, CSS, fonts, and images.
```
Comment (sidlall) [inline src/site/content/en/fast/signed-exchanges/index.md:324]: 'can also' or 'does also'?

File: src/site/content/en/fast/signed-exchanges/index.md [hunk 6]
```diff
@@ -325,6 +305,46 @@ Any site can cache, serve, and prefetch SXGs of the pages that it links to, wher
...
+- In the near future, SXG prefetching from Google Search will be available on more surfaces, including desktop form factors and all Chromium browsers.
+
+{% Details %}
```
Comment (sidlall) [inline src/site/content/en/fast/signed-exchanges/index.md:327]: Suggest deleting this section. IMO it seems speculative for the purpose of this discussion.

File: src/site/content/en/fast/signed-exchanges/index.md [hunk 7]
```diff
@@ -325,6 +305,46 @@ Any site can cache, serve, and prefetch SXGs of the pages that it links to, wher
...
+### Fewer requests to serve
+
+With cross-site prefetching, your server may need to serve additional requests. This corresponds to cases where a page was prefetched, but either the user didn't visit the page, or the prefetched bytes couldn't be shown to the user. For SXG, this additional traffic can be significantly reduced:
+
+- SXGs are cached and may be sent to users until they expire.
+- SXGs can be shown to users with cookies on your site.
+
+### Page speed improvement
+
+You may see additional page speed improvement from the following factors:
```
Comment (sidlall) [inline src/site/content/en/fast/signed-exchanges/index.md:321]:
```suggestion
In addition to this, Signed Exchanges has certain unique features that can additional provide page speed benefits for no incremental development or serving costs ``

File: src/site/content/en/fast/signed-exchanges/index.md (hunk 2 above)
Comment (twifkak) [reply src/site/content/en/fast/signed-exchanges/index.md]: OK, renamed to "additional unused requests". I worry "wasted" may be too negative -- no prediction is 100% precision.

File: src/site/content/en/fast/signed-exchanges/index.md (hunk 3 above)
Comment (twifkak) [reply src/site/content/en/fast/signed-exchanges/index.md]: OK, added longer explanations for both of the list items here.

File: src/site/content/en/fast/signed-exchanges/index.md (hunk 1 above)
Comment (twifkak) [reply src/site/content/en/fast/signed-exchanges/index.md]: Done.

File: src/site/content/en/fast/signed-exchanges/index.md (hunk 5 above)
Comment (twifkak) [reply src/site/content/en/fast/signed-exchanges/index.md]: Fixed. Does, when specified using a `Link` header. (Technically, a link tag is sufficient when using sxg-rs or webpkg...)

File: src/site/content/en/fast/signed-exchanges/index.md (hunk 6 above)
Comment (twifkak) [reply src/site/content/en/fast/signed-exchanges/index.md]: Yeah, OK, done.

File: src/site/content/en/fast/signed-exchanges/index.md (hunk 4 above)
Comment (twifkak) [reply src/site/content/en/fast/signed-exchanges/index.md]: OK, reframed the intro section to be more general. The specific two items here are current limitations of the Google SXG Cache, not of the SXG format. There are other potential improvements at the format level but they didn't make my top 2. :)

File: src/site/content/en/fast/signed-exchanges/index.md (hunk 7 above)
Comment (twifkak) [reply src/site/content/en/fast/signed-exchanges/index.md]: OK, tried to reword this a bit. PTAL.

**Review by twifkak — COMMENTED** [review]
Thanks for the feedback!

File: src/site/content/en/fast/signed-exchanges/index.md [hunk 8]
```diff
@@ -147,32 +147,16 @@ SXG works best in tandem with other performance optimizations such as use of CDN
 
 From past experiments we have observed an average of 300ms to 400ms reduction in LCP from SXG-enabled prefetches. This helps sites make a better first-impression on users and often has a positive impact on business metrics. 
 
-Several global brands and sites have already benefited from Signed Exchanges. As a case study, lets look at how implementing Signed Exchanges helped [RebelMouse](https://www.rebelmouse.com/signed-exchange), a prominent Content Management System (CMS), improve their customer’s performance and business metrics:
+Several global brands and sites have already benefited from Signed Exchanges. As a case study, lets look at how implementing Signed Exchanges helped [RebelMouse](https://www.rebelmouse.com/signed-exchange?utm_medium=post&utm_source=google&utm_campaign=signed-exchange-co-marketing&utm_content=signed-exchange), a prominent Content Management System (CMS), improve their customers’ performance and business metrics:
```
[the remark this answers is not in the document]
Comment (malchata) [reply src/site/content/en/fast/signed-exchanges/index.md:150]: Our style guide calls for straight quotes, but it's something that's hard to be consistent with. I'll mark these as I see them.

**Pull request merged**